common/main/eo.xml

```diff
@@ -1344,28 +1344,28 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				</dayPeriods>
 				<eras>
 					<eraNames>
-						<era type="0">↑↑↑</era>
+						<era type="0" draft="unconfirmed">antaŭ nia erao</era>
```

Comment:
same concern here, maybe `contributed`?

Reply:
If you make things unconfirmed or provisional, but others in the logical group are contributed or approved, that will cause a logical group failure.

Reply:
good point